devtools/inspect-assets.ipynb

```diff
@@ -50,61 +50,8 @@
     "\n",
     "from pudl.etl import defs\n",
     "\n",
-    "asset_key = \"exploded_balance_sheet_assets_ferc1\"\n",
```

Comment:
I removed some custom asset exploration work that got committed here. This is just suppose to be a template notebook.